torch/onnx/utils.py

```diff
@@ -508,6 +510,43 @@ def forward(self, x):
             All errors are subclasses of :class:`errors.OnnxExporterError`.
     """
 
+    if dynamo:
+        # Unsupported parameters for dynamo export
+        # TODO: These are not supported AT THE TIME
+        warnings.warn(
+            "f, export_params, verbose, training, input_names, output_names, operator_export_type, opset_version, "
+            "do_constant_folding, keep_initializers_as_inputs, custom_opsets, export_modules_as_functions, and "
+            "autograd_inlining are not supported for dynamo export at the moment."
+        )
+        # TODO: check args normalization
+        args = _decide_input_format(model, args)
+        kwargs = {}
```

Comment:
In a follow up PR, I would add an kwarg parameter to the api and encourage users to use that.

Reply:
torch.onnx.export has its own way to support kwargs. Do we want to add another way of supporting kwargs?
https://github.com/pytorch/pytorch/blob/4adee71155bec4e419bac32be2cbc1763bc6c98f/torch/onnx/utils.py#L240-L281